modules/combined/tests/fdp_geometric_coupling/fdp_geometric_coupling_test.i

```diff
@@ -211,4 +211,5 @@
   exodus = true
   print_linear_residuals = true
   print_perf_log = true
+  hide = 'dummy_name_contact_force_0 dummy_name_contact_force_1 dummy_name_contact_force_2'
```

Comment:
They are AuxVariables, but they do not use AuxKernels. Their values are managed from within MechanicalContactConstraint.